ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/disconnected/provision/cucushift-installer-rehearse-vsphere-ipi-disconnected-provision-chain.yaml

```diff
@@ -3,7 +3,10 @@ chain:
   steps:
   - chain: ipi-conf-vsphere
   - ref: ipi-install-rbac
-  - chain: vsphere-provision-bastionhost
+  - ref: ignition-bastionhost
+  - ref: ignition-bastionhost-httpd
+  - ref: vsphere-provision-bastionhost
+  - ref: ipi-conf-osimage
```

Comment:
Also need to update the following `cucushift-installer-rehearse-vsphere-ipi-disconnected-techpreview-provision-chain.yaml` ? 

And if vpshere-ipi-disconnected need this, does vsphere-ipi-proxy need this?

Reply:
yes, for vsphere-ipi-proxy, clusterOSImage setting is also needed if user don't export proxy on host where installer is launched.

Reply:
@jinyunma For our existing disconnected/proxy workflows in the upstream branch, we did not upload osimage and use them during installation, right? How we get the installation get passed?

Reply:
yes, because the running pod in ci cluster "vsphere" can connect to any subnetwork in VMC or IBMCloud.
The purpose of adding this step is that setting clusterOSImage in install-config.yaml is one test point if it is disconnected scenario and vm where run installer is also in this disconnected network.